rpcs3/Loader/ELF.h

```diff
@@ -190,6 +185,8 @@ class elf_object
 
 public:
 	elf_object() = default;
+	elf_object(const elf_object&) = delete; // Disabled for performance reasons
```

Comment:
What performance reasons? I don't see any. This object should be freely copyable or moveable.

Reply:
Okay.